scripts/vpn-port-guard.sh

```
@@ -1,361 +1,292 @@
+  local controller_mode="preferred"
+  local pf_enabled="false"
+  if bool_true "$pf_enabled_raw"; then
+    controller_mode="strict"
+    pf_enabled="true"
   fi
 
   local tmp
-  local tmp_template="${controller_state_dir}/state.XXXXXX"
-  if ! tmp="$(mktemp "${tmp_template}" 2>/dev/null)"; then
-    controller_log "Unable to create temporary state file"
+  if ! tmp="$(mktemp "${STATUS_DIR}/port-guard-status.XXXXXX" 2>/dev/null)"; then
```

Shellcheck (0.11.0)

[warning] Did you forget to close this double quoted string?

(SC1078)